cns/service/main.go

```diff
@@ -743,9 +743,12 @@ func InitializeMultiTenantController(httpRestService cns.HTTPService, cnsconfig
 	rootCxt := context.Background()
 	go func() {
 		// Periodically poll vfp programmed NC version from NMAgent
+		timerChannel := time.NewTicker(cnsconfig.SyncHostNCVersionIntervalMs * time.Millisecond).C
```

Comment:
usually we would not pull the channel out of the ticker, we would use a ticker var:
```
ticker := time.NewTicker(t)
defer ticker.Stop()
for {
        select {
        case <-ctx.Done():
            return
        case <- ticker.C:
            // work
}
```
although, since we have no need to ever stop the ticker here, we can abbreviate this using `time.Tick`:
```suggestion
		tick:= time.Tick(cnsconfig.SyncHostNCVersionIntervalMs * time.Millisecond)
```